shell/platform/windows/platform_view_manager.h

```diff
@@ -0,0 +1,52 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_SHELL_PLATFORM_WINDOWS_PLATFORM_VIEW_MANAGER_H_
+#define FLUTTER_SHELL_PLATFORM_WINDOWS_PLATFORM_VIEW_MANAGER_H_
+
+#include <memory>
+
+#include "flutter/shell/platform/common/client_wrapper/include/flutter/method_channel.h"
+#include "flutter/shell/platform/windows/public/flutter_windows.h"
+#include "flutter/shell/platform/windows/task_runner.h"
+
+namespace flutter {
+
+// Possible reasons for change of keyboard foucs.
+enum class FocusChangeDirection {
+  kProgrammatic,  // Un-directed focus change.
+  kForward,       // Keyboard focus moves forwards, e.g. TAB-key.
```

Comment:
I'm guessing the `-` was unintentional here:

```suggestion
  kForward,       // Keyboard focus moves forwards, e.g. TAB key.
```